index.html

```diff
@@ -75,62 +85,18 @@ <h3>Awards</h3>
                  <h3>Network</h3>
                  <p>Get opportunities to network with renown fashion icons and models and get useful insights from their pieces</p>
              </div>
+            </div>
+            <a href="" class="see-program hide">See the whole program</a>
              <button type="submit">Join OYA Summit</button>
         </div>
     </section>
     <section class="featured-speaker-section">
```

Comment:
The spacing/ alignment of your Featured Speakers sections does not match the provided design. Kindly note that we actually want this design to as much as possible mimic the provided  [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). Kindly ensure to improve upon the following:
- [ ] Add background decoration around each image
- [ ] Shapes of images should appear more of a square
- [ ] Ensure there are appropriate paddings around the elements of each content, as can be seen in the design template 
- [ ] Ensure that the contents are spaced appropriately, as can be seen in the design template:

**Your Featured speakers section:**
![image](https://user-images.githubusercontent.com/12745474/169597241-093f0a04-ea00-4eef-ab45-98f13c4b90dc.png)


**The original Featured speakers section:**
![image](https://user-images.githubusercontent.com/12745474/138319005-e3205012-0f6f-4535-ad6e-06f08229e3b9.png)